charts/gateway/README.md

```diff
@@ -838,6 +862,67 @@ ingress:
 | `pmtagger.containerSecurityContext`    | [Container Security Context](https://kubernetes.io/docs/tasks/configure-pod-container/security-context/#set-the-security-context-for-a-container)          | `{}` |
 
+### OpenTelemetry Configuration
+The Gateway from v11.1.00 can be configured to send telemetry to Observability backends [that support OpenTelemetry](https://opentelemetry.io/ecosystem/vendors/). Please see [Techdocs](https://techdocs.broadcom.com/us/en/ca-enterprise-software/layer7-api-management/api-gateway/11-1/install-configure-upgrade/configuring-opentelemetry-for-the-gateway.html) for more details about this integration.
+
+This feature is a ***preview feature*** for v11.1.00 and is ***intentionally disabled*** by default. As with any integration that generates telemetry, there is a performance drop when turning on the OpenTelemetry integration with all of the features enabled.
+
+There is an integration example avaialable [here](https://github.com/Layer7-Community/Integrations/tree/main/grafana-stack-prometheus-otel) that details how to deploy and configure an observability backend to use with the Gateway.
+- You are ***not required*** to use the observability stack that we provide as an example.
+- The observability stack that we provide ***is not*** production ready and should be used solely as an example or reference point.
+- OpenTelemetry is supported by [numerous vendors](https://opentelemetry.io/ecosystem/vendors/)
+
+***NOTE: *** In our example we inject the OpenTelemetry Java Agent to the Container Gateway, this emits additional telemetry like JVM metrics. The Gateway has the OpenTelemetry SDK built-in making the OpenTelemetry Java Agent Optional, the key difference between the built-in SDK and the OTel Agent is that the SDK only captures Gateway application level traces and metrics, things like JVM metrics will not be captured in this mode.
+
+#### Gateway OTel Configuration
+OpenTelemetry is configured on the Gateway in two places, system properties and cluster-wide Properties. The configuration below represents the minimal settings required to enable the built-in SDK and configure the Gateway to send telemetry to an OpenTelemetry Collector.
+
+These can be configured in values.yaml. See the section below to view examples of how and where to configure this.
+
+- system.properties
+```
+otel.sdk.disabled=false
+otel.java.global-autoconfigure.enabled=true
+otel.service.name=ssg-gateway
+otel.exporter.otlp.endpoint=http://localhost:4318/
+otel.exporter.otlp.protocol=http/protobuf
+otel.traces.exporter=otlp
+otel.metrics.exporter=otlp
+otel.logs.exporter=none
+```
+- cluster-wide properties
+```
+otel.enabled=true
+otel.serviceMetricEnabled=true
+otel.traceEnabled=true (if tracing is required)
+otel.traceConfig=(default {})
+```
+example otel.traceConfig
```

Comment:
For this section, may want to reference Techdocs to explain better the possible configuration values? Esp the excluding of assertions as it requires entering the assertion name (with the correct spacing) and where to find the name from Policy Manager.

Reply:
Link added under the section heading